daemon/src/db.rs

```diff
@@ -97,6 +97,13 @@ pub async fn load_order_by_id(
 }
 
 pub async fn insert_cfd(cfd: &Cfd, conn: &mut PoolConnection<Sqlite>) -> anyhow::Result<()> {
+    if load_cfd_by_order_id(cfd.order.id, conn).await.is_ok() {
```

Comment:
I thought I had introduced this check a long time ago...?
Weird.

Reply:
Would have been nice to have a separate commit for the db changes, I don't see how they are related to adding the projection actor and it would have been nice to have commit messages with explanations.

Reply:
mhm... that's in a separate commit with an explanation?